### PR TITLE
feat: Editable title & description of pr & issue and sticky issue title on scroll

### PR DESCRIPTION
**Pull request by pawanpaudel93**
## Summary

This PR covers the following things:

- Made pull request and issue description optional with editable title and description.
  - Editable title
image
  - Editable Description  
image

- Made issue title stick on scroll with (Close/Reopen) button and a "scroll to top" button on the left bottom.
image
- Prevent adding duplicate PRs between same branches
image
- Made markdown editors preview full height
- Fix some CSS issue with PR/Issue row in PR & Issue tab when title is longer.

- Resolved list style issue in Markdown editors.

### CODE REVIEW

File: src/helpers/getWrapContract.ts [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import { WarpFactory } from 'warp-contracts'
 import { DeployPlugin } from 'warp-contracts-plugin-deploy'
 
-const warp = WarpFactory.forMainnet().use(new DeployPlugin())
+// const warp = WarpFactory.forMainnet().use(new DeployPlugin())
+const warp = WarpFactory.forTestnet().use(new DeployPlugin())
```
Comment (7i7o) [inline src/helpers/getWrapContract.ts:5]: Will we merge this into main later?

File: warp/protocol-land/initial-state.json [hunk 2]
```diff
@@ -3,5 +3,5 @@
   "repos": {},
   "userRepoIdMap": {},
   "canEvolve": true,
-  "owner": "-OkLQwLgoND9TYEjcbQApQsAOlOmeWF5QBIIWn5DPaE"
+  "owner": "PykWdOZYg0DSmcHX4ts5zs-OJM71tjbR_YFaA-7Jn1k"
```
Comment (7i7o) [inline warp/protocol-land/initial-state.json:6]: Will this go into main later?

**Review by 7i7o — APPROVED** [review]
tACK! LGTM! 🚀

File: src/helpers/getWrapContract.ts (hunk 1 above)
Comment (pawanpaudel93) [reply src/helpers/getWrapContract.ts:5]: I am using a new deployed contract to testnet so i have to change back the contract address and warp to mainnet before we merge.

**Comment by 7i7o** [comment]
Really large .md description of issue breaks height when you want to preview it.
Otherwise it works great when the issue is created.

File: src/components/IssuePr/Description.tsx [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+          isIssue ? 'py-2' : 'py-[1.5px]'
+        } border-b-[1px] border-gray-300`}
+      >
+        {isIssue ? (
+          <div className="flex flex-auto justify-between">
+            <span>{shortenAddress(issueOrPr.author)}</span>
+            <span> {formatDistanceToNow(new Date(issueOrPr.timestamp), { addSuffix: true })}</span>
+          </div>
+        ) : (
+          <div className="font-medium">Description</div>
```
Comment (kranthicodes) [inline src/components/IssuePr/Description.tsx:65]: @pawanpaudel93 we lost the previous styles here when this new component replaced in PR read overview section

**Comment by pawanpaudel93** [comment]
> Really large .md description of issue breaks height when you want to preview it. Otherwise it works great when the issue is created.
I have removed the `Preview` button as the markdown can be previewed with the editor itself and not this issue should not be occuring.

File: src/components/IssuePr/Description.tsx (hunk 3 above)
Comment (pawanpaudel93) [reply src/components/IssuePr/Description.tsx:63]: It's resolved.

**Review by kranthicodes — APPROVED** [review]
tACK

**Pull request merged**